Sources/Linq2Acad/AcadDatabase.cs

```diff
@@ -10,6 +10,57 @@
 
 namespace Linq2Acad
 {
+  /// <summary>
+  /// DWG version enum.
+  /// </summary>
+  public enum SaveAsDwgVersion
+  {
+    None,
+    DWG2004,
+    DWG2007,
+    DWG2010,
+    DWG2013,
+#if AutoCAD_2018 || AutoCAD_2019 || AutoCAD_2020 || AutoCAD_2021
+    DWG2018,
+#endif
+    DontChange, // Better than Current? Or Keep? EDIT: We can use Database.LastSavedAsVersion for that
+    NewestAvailable // We could remove this one, since the newest available is AutoCad20xx wit hthe higehst number
+  }
+
+  public class CreateOptions
+  {
+    public bool SaveFileOnCommit { get; set; }
+
+    public string SaveAsFileName { get; set; }
```

Comment:
Maybe we should rename `SaveAsFileName` to `SaveFileName`, because we always save a newly created database for the first time. What do you think?

Reply:
It could also simply be `FileName`, since we `Create` the database it should be implicit that we will save it with that name. Or it is too implicit?

Reply:
What if we call this proberty `SaveFileName` and rename the property `DwgVersion` to `SaveFileDwgVersion`? Then the common prefix `SaveFile` would a) unambiguously identify what the properties are used for and would b) tell that the two properties go hand in hand.